dry-web.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "dry-component", "~> 0"
+  spec.add_runtime_dependency "dry-component"
```

Comment:
Yep, thanks 👍🏼 Will do that once I release the changes to dry-component. 

> On 16 Jun 2016, at 5:53 PM, Piotr Solnica <notifications@github.com> wrote:
> 
> In dry-web.gemspec:
> 
> > @@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
> >    spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
> >    spec.require_paths = ["lib"]
> >  
> > -  spec.add_runtime_dependency "dry-component", "~> 0"
> > +  spec.add_runtime_dependency "dry-component"
> are we sure we don't want a version spec?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>